tethys_gizmos/static/tethys_gizmos/css/tethys_map_view.css

```diff
@@ -158,6 +158,11 @@
   -ms-transition: height 0.2s ease-in;
   transition: height 0.2s ease-in;
 }
+.tethys-map-view-legend .legend-content .legend-items .legend-item .tethys-mvlegendimage.open {
```

Comment:
OK. Do we want to make the CSS minified to prevent future confusion? And it will probably be faster.

$ lessc --clean-css tethys_map_view.less tethys_map_view.min.css